FP_layout/app/src/main/java/com/example/dhyatmika/fp_layout/Helper.java

```diff
@@ -0,0 +1,103 @@
+package com.example.dhyatmika.fp_layout;
+
+import android.content.Context;
+import android.content.SharedPreferences;
+import android.content.res.Resources;
+import android.graphics.Bitmap;
+import android.graphics.drawable.BitmapDrawable;
+import android.widget.Toast;
+
+import com.android.volley.AuthFailureError;
+import com.android.volley.NetworkResponse;
+import com.android.volley.ParseError;
+import com.android.volley.Request;
+import com.android.volley.Response;
+import com.android.volley.VolleyError;
+import com.android.volley.toolbox.HttpHeaderParser;
+import com.android.volley.toolbox.JsonObjectRequest;
+import com.android.volley.toolbox.Volley;
+
+import org.json.JSONException;
+import org.json.JSONObject;
+
+import java.io.UnsupportedEncodingException;
+import java.lang.reflect.Method;
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.Map;
+
+public final class Helper {
+
+    private Helper() {
+        /*
+         *    private constructor to prevent instantiation of this class
+         */
+    }
+
+    public static final void toastLong(Context ctx, String message) {
```

Comment:
I would say having one method for toast would be simpler.

toast(ctx, message, 0)  // 0 short, 1 long, for example